### PR TITLE
Fix compilation error

### PR DESCRIPTION
**Pull request by i-stam**

### CODE REVIEW

File: contracts/ReversibleICO.sol [hunk 1]
```diff
@@ -682,7 +682,7 @@ contract ReversibleICO is IERC777Recipient {
     */
     function hasPendingETH(address _participantAddress) public view returns (bool) {
         Participant storage participantRecord = participantsByAddress[_participantAddress];
```
Comment (mickys) [inline contracts/ReversibleICO.sol]: We can have something like:
```
Participant storage participantTotals = participantsByAddress[_participantAddress].byStage[0];
if(participantTotals.committedETH > 0 && participantTotals.committedETH > participantTotals.returnedETH ) {
    return true;
}
```

File: contracts/ReversibleICO.sol [hunk 2]
```diff
@@ -1031,7 +1031,7 @@ contract ReversibleICO is IERC777Recipient {
 
                         // update return values
                         returnedETH = returnedETH.add(returnValue);
-                        participantRecord.returnedETH = participantRecord.returnedETH.add(returnValue);
+                        participantRecord.byStage[0].returnedETH = participantRecord.byStage[0].returnedETH.add(returnValue);
```
Comment (mickys) [inline contracts/ReversibleICO.sol]: same as above

and where we use byStage in a loop just assign from ` byStage 
 = participantsByAddress[_participantAddress].byStage[i]; `

**Comment by frozeman** [comment]
@i-stam you probably need to push another empty commit to restart travis.
Funny enough the build passes, but the tests fail and it doesnt show as error: https://travis-ci.org/lukso-network/rICO-smart-contracts/jobs/639405792

File: contracts/ReversibleICO.sol [hunk 3]
```diff
@@ -617,7 +618,7 @@ contract ReversibleICO is IERC777Recipient {
         uint256 stageReturnedTokens
     ) {
 
-        ParticipantDetailsByStage storage totalsRecord = participantsByAddress[_address].byStage[_stageId];
+        ParticipantDetails storage totalsRecord = participantsByAddress[_address].byStage[_stageId];
```
Comment (frozeman) [inline contracts/ReversibleICO.sol]: Is this still relevant if we now use `participantAggregatedStats[_participantAddress]`?
Also we have remove `participantsByAddress[_address].byStage`

Comment (i-stam) [reply contracts/ReversibleICO.sol]: I think this is for returning only the token related stats and it is used mainly for testing, hence the naming in the return arguments, maybe @mickys  can help us on that. Are your referring about removing completely .byStage[_stageId]. I think if we really want it we can  think of a smart way to keep track of all stages, I can think about it more.

Comment (i-stam) [reply contracts/ReversibleICO.sol]: Since the struct is public, this function it not needed in a strict way

**Pull request merged**